chain-impl-mockchain/src/ledger/ledger.rs

```diff
@@ -497,6 +497,7 @@ impl Ledger {
                 Fragment::EncryptedVoteTally(_) => {
                     return Err(Error::Block0(Block0Error::HasVoteTally));
                 }
+                Fragment::SmartContractDeploy(_) => todo!(),
```

Comment:
So, this is the next frontier. What we want is to initialize the ledger state with the smart contract data contained in the fragment.

Reply:
Cool. In order to get this working, I'm adding EVM params to the block's Config types.